### PR TITLE
Fix st-tool bug and improve documentation

### PR DESCRIPTION
**Pull request by adrianbroher**
This PR fixes a bug, which blocks the merge of #2851 and applies suggestions made in #2844.

### CODE REVIEW

**Comment by Oberlus** [comment]
Once this is merged, the TravisCI won't fail for #2851, right?
And to continue with the translation I can keep using the st-tool in #2844, no need for me to pull this branch for that purpose, isn't it?

**Comment by adrianbroher** [comment]
> Once this is merged, the TravisCI won't fail for #2851, right?

You would have to rebase the branch onto the merge commit or a later commit.

> And to continue with the translation I can keep using the st-tool in #2844, no need for me to pull this branch for that purpose, isn't it?

That's correct.

File: check/st-tool.py [hunk 1]
```diff
@@ -542,39 +544,91 @@ def compare_action(args):
...
+        * Translation entries (translated or not) are separated by one empty
+          line.
+        * Translation notes (prefix: '# ') and untranslated entries
+          (prefix: '#*') are properly prefixed and formatted
+        * Translation notes, key and value don't have empty lines between
+          each other.
+        * String tables contain the language name as first line, followed by
+          the file notes.
+        * Section titles are prefixed with a block comment (prefix '##').
+        * Section titles have two leading whitespaces and one trailingi
```
Comment (geoffthemedio) [inline check/st-tool.py]: extra i on trailing

**Comment by geoffthemedio** [comment]
Do you want this tested or other feedback, or just for me to click merge for you...?

**Comment by adrianbroher** [comment]
> Do you want this tested or other feedback, or just for me to click merge for you...?

The script should work fine (apart from the typo). I cannot merge.

**Pull request merged**